logger.go

```diff
@@ -67,10 +74,19 @@ func (logger *Logger) Copy(module string) *Logger {
 	for key, hook := range logger.hooks {
 		hooks = append(hooks, NewHook(key, hook.level, hook.hook.inner))
 	}
-	newLogger := New(module, hooks...)
+	newLogger := *logger
+	newLogger.hooks = nil
+	newLogger.AddHooks(hooks...)
 	newLogger.Entry = logger.Entry.WithField(moduleFieldName, module)
-	newLogger.PrintLevel = logger.PrintLevel
-	return newLogger
+	return &newLogger
+}
+
+// Child returns a new logger with the same module and properties as its parent.
```

Comment:
Not the same module actually. 
`Child clones the logger, appending the child's name to the parent's module name`

Reply:
Done